pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd_migration_use-cases/guide.pt-pt.md

```diff
@@ -61,11 +61,11 @@ The table below lists each of the migration bottlenecks and criticality levels t
-| **Step 7**  | ⚠️           | **Resource pools (sharing)**            | No solutions for now.                                                                                                                                        | - Resource pools will be lost after migration because this notion no longer exists on the VCD side. We recommend the use of the vApp concepts in the VCD control panel.                                                | [Working with vApps in the VCD on OVHcloud control panel](<https://docs.vmware.com/en/VMware-Cloud-Director/10.6/VMware-Cloud-Director-Tenant-Guide/GUID-AC48FB5E-4ADC-4835-AACE-B949B297A147.html>) |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               
+| **Step 7**  | ⚠️           | **Resource pools (sharing)**            | No solutions for now.                                                                                                                                        | - Resource pools will be lost after migration because this notion no longer exists on the VCD side. We recommend the use of the vApp concepts in the VCD control panel.                                                | [Working with vApps in the VCD on OVHcloud control panel](<https://docs.vmware.com/en/VMware-Cloud-Director/10.5/VMware-Cloud-Director-Tenant-Guide/GUID-AC48FB5E-4ADC-4835-AACE-B949B297A147.html>) |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               
```

Comment:
Fix this link

Reply:
[Working with vApps in the VCD on OVHcloud control panel](https://docs.vmware.com/en/VMware-Cloud-Director/10.5/VMware-Cloud-Director-Tenant-Guide/GUID-AC48FB5E-4ADC-4835-AACE-B949B297A147.html) @tcpdumpfbacke ?

Reply:
https://www.markdownguide.org/basic-syntax/#links